config/jobs/periodic/containerd/test-containerd-powervs-periodic.yaml

```diff
@@ -34,7 +34,7 @@ periodics:
 
               # Run test server
               mkdir ${ARTIFACTS}/runc_l1/
-              bash -x ./instantiate_vm_and_test.sh --output ${ARTIFACTS}/runc_l1/ --name rdr-runtimes-containerd-runc-l1 --runc runc --runtime "io.containerd.runtime.v1.linux" --key atos-key --network  'public-192_168_139_176-29-VLAN_2050' 2>&1 | tee runc_l1.txt
+              bash -x ./instantiate_vm_and_test.sh --output ${ARTIFACTS}/runc_l1/ --name rdr-runtimes-containerd-runc-l1 --runc runc --runtime "io.containerd.runtime.v1.linux" --key atos-key --network  'public-192_168_139_0-29-VLAN_2028' 2>&1 | tee runc_l1.txt
```

Comment:
@Jenkins-J 
Ref: https://github.com/ppc64le-cloud/docker-ce-build/pull/78
Looks like a new public network will be created for new PowerVS instances at the start of each test. New PowerVS instances will no longer depend on an existing public network.

Can we stop sending the network name here to the `--network` flag? also, you may remove the network flag from `instantiate_vm_and_test.sh` as it happens to create a new network every time an instance is created.

Saw a new network by name `prow-net-3EXRhjnxmI` being created inspite of passing `--network` in the job definition for `periodic-containerd-runc-l1-test-powerVS`
[link to job](https://s3.us-south.cloud-object-storage.appdomain.cloud/ppc64le-prow-logs/logs/periodic-containerd-runc-l1-test-powerVS/1630904474371362816/build-log.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=d5b0d1e12bb74ec5b0cca25bec3e0746%2F20230302%2Fus-south%2Fs3%2Faws4_request&X-Amz-Date=20230302T073145Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Signature=b9cc757431b790898af40d7432f0ebe8e7b40e67de8442ee94fd9cc3862ca0cb)

Reply:
You are correct, I forgot to update this PR to match those changes. I will remove the `--network` flag.

Reply:
@Rajalakshmi-Girish I have removed the network flag.